hack/generator/pkg/codegen/pipeline_load_schema.go

```diff
@@ -45,6 +46,40 @@ func (factory *cancellableJSONLoaderFactory) New(source string) gojsonschema.JSO
 	return &cancellableJSONLoader{factory.ctx, factory.inner.New(source)}
 }
 
+type rewritingJSONLoader struct {
+	from, to string
+	inner    gojsonschema.JSONLoader
+}
+
+var _ gojsonschema.JSONLoader = &rewritingJSONLoader{}
+
+func (loader *rewritingJSONLoader) LoadJSON() (interface{}, error) {
+	return loader.inner.LoadJSON()
+}
+
+func (loader *rewritingJSONLoader) JsonSource() interface{} {
+	return loader.inner.JsonSource()
+}
+
+func (loader *rewritingJSONLoader) JsonReference() (gojsonreference.JsonReference, error) {
+	return loader.inner.JsonReference()
+}
+
+func (loader *rewritingJSONLoader) LoaderFactory() gojsonschema.JSONLoaderFactory {
+	return &rewritingJSONLoaderFactory{loader.from, loader.to, loader.inner.LoaderFactory()}
+}
+
+type rewritingJSONLoaderFactory struct {
+	from, to string
+	inner    gojsonschema.JSONLoaderFactory
+}
+
+var _ gojsonschema.JSONLoaderFactory = &rewritingJSONLoaderFactory{}
+
+func (factory *rewritingJSONLoaderFactory) New(source string) gojsonschema.JSONLoader {
+	return factory.inner.New(strings.Replace(source, factory.from, factory.to, 1))
```

Comment:
At first I thought source was the contents of the schema, but rereading it's the URL, right?

Reply:
'Cause the former would be pretty blunt! Although I guess it would work, so :shrug: still would've been better than what we have now.